### DISCUSSION TITLE
owid-covid-data.csv has not been updated for 1 week

### DISCUSSION THREAD

**Discussion by ElsieZhang** [owid/covid-19-data#2838] [General]
owid-covid-data.csv has not been updated for 1 week. Will this dataset be discontinued?

**Comment by lucasrodes** [comment]
Hi @ElsieZhang,
Please use our recommended stable URL: https://covid.ourworldindata.org/data/owid-covid-data.csv.

Since files are large (exceed 100MB), it is no longer possible to update them on Github.
